src/main.rs

```diff
@@ -1,5 +1,6 @@
 #![allow(unknown_lints)] // for clippy
 
+#[macro_use] extern crate itertools;
```

Comment:
You can remove the `#[macro_use]` here, that will fix the clippy CI failure.

Reply:
But this line is needed. It does not compile when i remove it. Fix is probably to move all the code to new edition and fix all warnings. I can do that, thow it feels like it should be a new PR

Reply:
The `extern crate` is still needed, but it doesn’t need the `#[macro_use]` on it.